### ISSUE TITLE
continuous integration for appveyor and travis

### ISSUE THREAD

**Issue by den-run-ai** [peteut/ipython-file-upload#10]
Without CI and some basic tests, it is hard to believe that something is working in this extension.

At least people can look at YAML and installation logs to see all dependencies, which are complicated due to distributed jupyter ecosystem.

I will try to run this on linux mint with chrome and mozilla browsers.

**Comment by peteut** [comment]
@denfromufa: Agreed, do you know a project where I can learn how to use Travis CI to test the browser side?

**Comment by den-run-ai** [comment]
@peteut this is how testing for one of nbextensions works:

https://github.com/ipython-contrib/IPython-notebook-extensions/tree/master/nbextensions/usability/highlighter#testing

**Comment by den-run-ai** [comment]
not sure how it can be run in CI though.

**Comment by peteut** [comment]
`karma` is used in [jupyter-js-services/test/integration_test.py](https://github.com/jupyter/jupyter-js-services/blob/master/test/integration_test.py)

See also https://github.com/jupyter/notebook/issues/586
